packages/next-server/server/next-server.js

```diff
@@ -194,8 +194,7 @@ export default class Server {
 
   async renderToHTML (req, res, pathname, query) {
     try {
-      const out = await renderToHTML(req, res, pathname, query, this.renderOpts)
-      return out
+      return await renderToHTML(req, res, pathname, query, this.renderOpts)
```

Comment:
```suggestion
      return renderToHTML(req, res, pathname, query, this.renderOpts)
```

`return await` is also redundant 😄

Reply:
Removed `return await`!

Reply:
The suggestion is to remove `await` `return` is still needed

Reply:
Do we need `async` here if we are not using `await`?

Should it be something like:

```
  renderToHTML (req, res, pathname, query) {
    try {
      return renderToHTML(req, res, pathname, query, this.renderOpts)
```